service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -33,7 +33,8 @@ public class MigrateCaseController extends CallbackController {
 
     private final Map<String, Consumer<CaseDetails>> migrations = Map.of(
         "DFPL-log", this::runLog,
-        "DFPL-2527", this::run2527
+        "DFPL-2527", this::run2527,
```

Comment:
Same question - is there a reason why you've left DFPL-2527 in? :)

Reply:
hmm... no reason, just didn't check 2527 is done already. XD
I have removed scripts that already done.